cfme/tests/webui/test_general_ui.py

```diff
@@ -567,6 +581,67 @@ def test_infrastructure_provider_left_panel_titles(
         soft_assert(test_view.is_displayed, f"{test_view} not displayed.")
 
 
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.parametrize("provider", PROVIDERS)
+@pytest.mark.meta(automates=[1741030, 1783208])
+def test_provider_documentation(temp_appliance_preconfig, provider, request):
+    """
+    Bugzilla:
+        1741030
+        1783208
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/18h
+        startsin: 5.11
+        setup:
+            1. Take a fresh appliance with no provider
+        testSteps:
+            1. Log into the appliance, navigate to the provider All page
+                and check where the anchor link provided in
+                `Learn more about this in the documentation.` points to.
+        expectedResults:
+            1. Link must point to downstream documentation and not upstream.
+    """
+    url = (
+        "https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html"
+        f"/managing_providers/{PROVIDERS[provider]}"
+    )
+    destination = "AllOfType" if provider in (AnsibleTowerProvider, SatelliteProvider) else "All"
+
+    view = navigate_to(provider, destination)
+
+    initial_count = len(view.browser.window_handles)
+    main_window = view.browser.current_window_handle
+
+    href = Text(
```

Comment:
Consider defining a widget on the Provider view instead of just putting this in the test.